cmake/modules/FindUV.cmake

```diff
@@ -22,14 +22,16 @@ find_path(
 )
 
 find_library(
-    UV_STATIC_LIBRARY NAMES libuv.a
-    PATHS ${BUILD_DEPS_DIR}/${UV_DEPS_DIR}/.libs/
+    UV_STATIC_LIBRARY NAMES libuv
```

Comment:
I thought it was the other way around.
The Travis build reports for the pull request show that they chose `libuv.a` to represent UV on both [precise](https://travis-ci.org/skypjack/uvw/jobs/162603519#L609) and [macOS](https://travis-ci.org/skypjack/uvw/jobs/162603520#L310).
Also, the tests passed afterwards, which wouldn't have on macOS due to lack of rpath configuration.